data/json/items/ammo/cannon.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "cannon_shot",
+    "type": "AMMO",
+    "name": { "str": "cannon round shot" },
+    "description": "A bundle of wadding, black gunpowder, and cannonball, ready to be loaded into a cannon",
+    "weight": "4800 g",
+    "volume": "2500 ml",
+    "price": 11000,
+    "price_postapoc": 12000,
```

Comment:
$120? For a few lbs of lead and gunpowder?  That's pretty high.  $12 is probably more reasonable.